### ISSUE TITLE
Incorrect [Publication year] on Datacite metadata

### ISSUE THREAD

**Issue by neil-ices-dk** [ices-tools-dev/DOI#6]
The DataCite record is showing an incorrect year, in the ICES metadata 
http://www.ices.dk/sites/pub/Publication%20Reports/Forms/DispForm.aspx?ID=34097 
the year is '2018' and in the DataCite it is '2016' 
https://search.datacite.org/works?query=10.17895%2Fices.pub.4069

**Comment by Nas-ICES-IT** [comment]
It has been fixed now, But DOI created before we need to find a way to update them.

**Comment by neil-ices-dk** [comment]
@HansMJ i assume you will need to push them back out from the localDB?

**Comment by neil-ices-dk** [comment]
@Nas-ICES-IT 

- [ ] to make [Publication Author] mandatory, and make a default text entry 'ICES'
- [ ] re-extract metadata from Sharepoint (and default author to be added)

@mikeydrew 

- [ ] to force the udpate to Datacite using their 'new' services for all 327 DOI records
- [ ] make the test DOI's inactive on Datacite

**Comment by neil-ices-dk** [comment]
Celine has created DOI with a latest changes in term of Data mapping: 

Here is the metadata in XML: 
https://api.datacite.org/works/10.17895/ices.pub.4370 

In GUI: 

https://search.datacite.org/works?query=10.17895%2Fices.pub.4370

**Comment by neil-ices-dk** [comment]
see #9

**Comment by Nas-ICES-IT** [comment]
Schema look like this: 
![image](https://user-images.githubusercontent.com/38248617/40364316-4ebd9468-5dd2-11e8-94e4-3a58369ebb88.png)

**Comment by mikeydrew** [comment]
This is how it looks in the new search API ->
![image](https://user-images.githubusercontent.com/10516840/40421393-0e11afd4-5e8c-11e8-8a86-fa76a563b12d.png)

**Comment by neil-ices-dk** [comment]
discussed 23/05 - still needs testing

**Comment by mikeydrew** [comment]
With the new API ->
https://api.datacite.org/works/10.17895/ices.pub.4373

![image](https://user-images.githubusercontent.com/10516840/40488528-470cc1d0-5f67-11e8-8b9f-6657862113d5.png)

**Comment by neil-ices-dk** [comment]
@Nas-ICES-IT 

- [x] to make [Publication Author] mandatory, and make a default text entry 'ICES'
- [ ] re-extract metadata from Sharepoint (and default author to be added)
@CelineByrne can your intern help with the above task?

@mikeydrew 

- [ ] to force the udpate to Datacite using their 'new' services for all 327 DOI records
- [ ] make the test DOI's inactive on Datacite

**Comment by neil-ices-dk** [comment]
@CelineByrne will this be ok to start after the ASC, is that the plan

**Comment by CelineByrne** [comment]
@Nas-ICES-IT @neil-ices-dk @mikeydrew yes Silvia is ready to take on these tasks - please let me know how soon she can start as she is only here until the end of October
